src/components/UI/molecules/AuctionCard/index.module.scss

```diff
@@ -11,6 +11,15 @@
   break-inside: avoid;
   max-width: 100%;
 
+  .memb {
```

Comment:
What does `memb` mean? Is this supposed to be `menu`?

Reply:
It is the meaning of membrane.